projects/igniteui-angular/src/lib/grids/hierarchical-grid/hierarchical-grid-base.directive.ts

```diff
@@ -158,12 +160,12 @@ export abstract class IgxHierarchicalGridBaseDirective extends IgxGridBaseDirect
         zone: NgZone,
         @Inject(DOCUMENT) public document,
         cdr: ChangeDetectorRef,
-        resolver: ComponentFactoryResolver,
         differs: IterableDiffers,
         viewRef: ViewContainerRef,
         appRef: ApplicationRef,
         moduleRef: NgModuleRef<any>,
         injector: Injector,
+        protected envInjector: EnvironmentInjector,
```

Comment:
Spare me some work after conflict resolutions and don't override the base property please 
```suggestion
        envInjector: EnvironmentInjector,
```
See https://github.com/IgniteUI/igniteui-angular/pull/12856 for explanation :)